app/controllers/admins/employees_controller.rb

```diff
@@ -16,15 +16,31 @@ def edit
 
     def update
       if employee.update(employee_params)
-        redirect_to admins_employee_path(employee), notice: 'Employee was successfully updated.'
+        redirect_to admins_employee_path(employee), info: 'Employee was successfully updated.'
       else
         render :edit, locals: { employee: employee }
       end
     end
 
     def destroy
       employee.destroy
-      redirect_to admins_employees_path, notice: 'Employee was successfully destroyed.'
+      redirect_to admins_employees_path, danger: 'Employee was successfully destroyed.'
+    end
+
+    def kudos_for_all
+      render :update_kudos_for_all, locals: { employees: Employee.all }
+    end
+
+    def update_kudos_for_all
+      if Employee.all.each do |e|
```

Comment:
what if an error will be thrown at line 38 given you're in the middle of each loop? I'll tell you - you will have messed up data :) I'd think about using a transaction here.

the second thing is this `if` from line 35. Did you test if it work? :)